Dockerfile

```diff
@@ -44,7 +44,8 @@ RUN go build --tags musl -o rosetta .
 #---------------------------------------------------------------------
 
 # adbdc7f8c27e50e77b407cb328193bd3ee823643 is mainet(1.0.0) + a few compatible commits that include tracing fix
```

Comment:
I am unsure whether the 1.1.1 release has the tracing fix mentioned here.

Reply:
Confirming that this fix ([here](https://github.com/celo-org/celo-blockchain/pull/989)) is included in 1.1.1